adam-r/bdgenomics.adam/R/rdd.R

```diff
@@ -19,44 +19,75 @@ library(SparkR)
 
 setOldClass("jobj")
 
+#' A class that wraps an RDD of genomic data with helpful metadata.
+#'
+#' @rdname GenomicRDD
+#' @slot jrdd The Java RDD that this class wraps.
+#' 
 #' @export
 setClass("GenomicRDD",
          slots = list(jrdd = "jobj"))
 
-
+#' A class that wraps DataFrame of genomic data with helpful metadata.
+#'
+#' @rdname GenomicDataset
+#' @slot jrdd The Java RDD that this class wraps.
+#' 
 #' @export
 setClass("GenomicDataset",
          slots = list(jrdd = "jobj"),
          contains = "GenomicRDD")
 
-
+#' A class that wraps an RDD of genomic reads with helpful metadata.
+#'
+#' @rdname AlignmentRecordRDD
+#' @slot jrdd The Java RDD of AlignmentRecords that this class wraps.
+#' 
 #' @export
 setClass("AlignmentRecordRDD",
          slots = list(jrdd = "jobj"),
          contains = "GenomicDataset")
 
+#' @importFrom methods new
 AlignmentRecordRDD <- function(jrdd) {
     new("AlignmentRecordRDD", jrdd = jrdd)
 }
 
+#' A class that wraps an RDD of genomic coverage data with helpful metadata.
+#'
+#' @rdname CoverageRDD
+#' @slot jrdd The Java RDD of Coverage that this class wraps.
+#' 
 #' @export
 setClass("CoverageRDD",
          slots = list(jrdd = "jobj"),
          contains = "GenomicDataset")
 
+#' @importFrom methods new
 CoverageRDD <- function(jrdd) {
     new("CoverageRDD", jrdd = jrdd)
 }
 
+#' A class that wraps an RDD of genomic features with helpful metadata.
+#'
+#' @rdname FeatureRDD
+#' @slot jrdd The Java RDD of Features that this class wraps.
+#' 
 #' @export
 setClass("FeatureRDD",
          slots = list(jrdd = "jobj"),
          contains = "GenomicDataset")
 
+#' @importFrom methods new
 FeatureRDD <- function(jrdd) {
     new("FeatureRDD", jrdd = jrdd)
 }
 
+#' A class that wraps an RDD of read fragments with helpful metadata.
```

Comment:
I find "read fragments" to be a bit confusing, but I suppose we use that elsewhere.  "Read pairs grouped by sequencing fragment" is probably too verbose?